partition/roles/mgmt-server/tasks/main.yaml

```diff
@@ -39,11 +39,26 @@
     name:
     - net-tools
     - ipmitool
-    - docker.io
     - iptables-persistent
     update_cache : yes
     force_apt_get: yes
 
+- name: install docker
```

Comment:
Unfortunately this is breaking for existing management server deployments.

One idea would be to figure out if docker is already installed an then skip this step.